README.md

```diff
@@ -1,6 +1,133 @@
+
+1. After the owner of the `CTI-website-frontend` repository approves and merges your Pull Request, your changes will be live on the website. 
+
+#### Step 2 Fork the repository
+
+In https://github.com/CTI-website-frontend/website, look for the fork icon in the top right. Click it and create a fork of the repository.
+
+For git beginners, a fork is a copy of the repository that will be placed on your GitHub account url.
+
+It should create a copy here: https://github.com/your_GitHub_user_name/website, where `your_GitHub_user_name` is replaced with exactly that.
+
+Note that this copy is on a remote server on the GitHub website and not on your computer yet.
+
+If you click the icon again, it will not create a new fork but instead give you the URL associated with your fork.
+
+#### Step 3 Clone your online repository to your local computer
+
+For git beginners, this process will create a third copy of the repository on your local desktop.
+
+First create a new folder on your desktop that will contain `CTI-website-frontend` projects.
+
+In your shell, navigate there then run the following commands:
+
+```bash
+git clone https://github.com/your_GitHub_user_name/website.git
+```
+
+You should now have a new folder in your `CTI-website-frontend` folder called `website`.
+
+Verify which URL your `origin` remote is pointing to:
+
+```bash
+git remote show origin
+```
+
+If you accidentally cloned the `CTI-website-frontend/website.git` then you can correct that with the following two commands: 
+
+1) Change your local copy to upload to your fork with the following:
+
+```bash
+git remote set-url origin https://github.com/your_user_name/website.git
+```
+
+2) Add another remote called `upstream` that points to the `CTI-website-frontend` version of the repository. This will allow you to incorporate changes later:
+
+```bash
+git remote add upstream https://github.com/CTI-website-frontend/website.git
+```
+
+#### Step 4: Setting up Docker
+
+Docker is the recommended approach to quickly getting started with local development. (ELI5: Docker helps create a local/offline version of the CTI-website-frontend.org website on your computer so you can test out your code before submitting a pull request).
+
+There are two pre-requisites: Docker and Docker Compose.
+The recommended installation method is [Docker Desktop](https://docs.docker.com/install/) for Windows 10 64-bit,
+Mac, and Linux users. Users of unsupported operating systems may check out [Docker Toolbox](https://docs.docker.com/compose/gettingstarted/) instead.
+
+More on using Docker and the concepts of containerization:
+
+* [Get started with Docker](#docker)
+* [Get started with Docker Compose](https://docs.docker.com/compose/gettingstarted/)
+
+*Ensure you run the `docker` commands below from a shell inside the local directory containing your clone of this repository.*
+
+
```

Comment:
The Docker run instructions below are from the original README.md file, the ones above are from hackforla